spec/language_server/definition_spec.cr

```diff
@@ -0,0 +1,45 @@
+require "../spec_helper"
+
+def clean_json(workspace : Workspace, path : String)
+  path.strip.gsub("\#{root_path}", workspace.root_path)
+end
+
+Dir
+  .glob("./spec/language_server/definition/**/*")
+  .select! { |file| File.file?(file) }
+  .sort!
+  .each do |file|
+    it file do
+      with_workspace do |workspace|
+        manifest, *rest = File.read(file).split("-" * 80)
```

Comment:
Oh, this is neat, didn't know Crystal supports this :metal:

Reply:
Will probably change the tests to match the [type_checking ones](https://github.com/mint-lang/mint/blob/master/spec/type_checking/html_element_not_in_component) rather than having a manifest at the top for consistency, so this will might have to go 😅 

Have been enjoying my time with Crystal so far tho!